spyder/preferences/shortcuts.py

```diff
@@ -470,6 +475,13 @@ def unbind_shortcut(self):
         self._qsequences = []
         self.accept()
 
+    def accept_override(self):

```

Comment:
Please add a docstring to this method.

Reply:
Done.

Reply:
Please add a period to the end

like `"""Unbind all conflicted shortcuts, and accept the new one."""`

https://www.cockos.com/licecap/

Reply:
I've added a period, but seems too late. Sorry.

Reply:
Opps xD